src/index.js

```diff
@@ -22,10 +22,10 @@ if (
 }
 
 ReactDOM.render(
-  <Main>
-    <GlobalErrorHandler>
+  <GlobalErrorHandler>
```

Comment:
I thought you would integrate `GlobalErrorHandler ` into `Main`

Reply:
Oh I see, no `GlobalErrorHandler` is really an aragon/aragon component ; aragonUI lets users handle errors the way they want in their app.